Documentation/git-refs.txt

```diff
@@ -39,6 +43,13 @@ include::ref-storage-format.txt[]
 	can be used to double check that the migration works as expected before
 	performing the actual migration.
 
+--strict::
+	Enable more strict checking, every WARN severity will be seen as
+	ERROR.
```

Comment:
What does WARN and ERROR severity here refer to? It would be nice to elaborate a bit more on the levels.

Reply:
There are some types of severity for `git-fsck(1)` messages defineed in `fsck.h`. I will elaborate this.